public/app/features/transformers/editors/OrganizeFieldsTransformerEditor.tsx

```diff
@@ -19,10 +19,11 @@ import { useAllFieldNamesFromDataFrames } from '../utils';
 interface OrganizeFieldsTransformerEditorProps extends TransformerUIProps<OrganizeFieldsTransformerOptions> {}
 
 const OrganizeFieldsTransformerEditor = ({ options, input, onChange }: OrganizeFieldsTransformerEditorProps) => {
-  const { indexByName, excludeByName, renameByName } = options;
+  const { indexByName, excludeByName, renameByName, includeByName } = options;
 
   const fieldNames = useAllFieldNamesFromDataFrames(input);
   const orderedFieldNames = useMemo(() => orderFieldNamesByIndex(fieldNames, indexByName), [fieldNames, indexByName]);
+  const filterType = includeByName ? 'include' : 'exclude';
```

Comment:
includeByName is defaulting to `{}` which makes the filterType default to include. To preserve the current functionality I think the filterType should default to exclude.

Reply:
Good catch, should be fixed now!